### PR TITLE
FP-3374: Fixed versioning for deploy

### PR DESCRIPTION
**Pull request by robhuzzey**
#### What does this PR do? (please provide any background)
Fixes the versioning for a change to legoland.

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### Any tech debt?
no

#### Screenshots / Screencast
N/A

#### What gif best describes how you feel about this work?
Nope.

- [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "brands",
-  "version": "1.0.2",
+  "version": "1.0.1",
```
Comment (jodiedoubleday) [inline package.json]: do not change this version!

Comment (jodiedoubleday) [reply package.json]: this should never have been changed, please return back to 1.0.0

Comment (robhuzzey) [reply package.json]: it was at 1.0.1

Comment (jodiedoubleday) [reply package.json]: ahh okay (wonder who did that)

**Pull request merged**